modules/scanning-images.adoc

```diff
@@ -58,80 +58,96 @@ IMPORTANT: If you are using a Red Hat link:https://catalog.redhat.com/software/c
 
 * .NET Core
 * ASP.NET Core
 
-[discrete]
+These are not supported by Scanner V4.
+
+[id="supported-operating-systems_{context}"]
 == Supported operating systems
 
 The supported platforms listed in this section are the distributions in which Scanner identifies vulnerabilities, and it is different from the supported platforms on which you can install {product-title}.
 
 Scanner identifies vulnerabilities in images that contain the following Linux distributions:
 
 |===
-| Distribution | Version
+| Distribution | Version | Vulnerability database
 
 | link:https://www.alpinelinux.org/[Alpine Linux]
-| `alpine:v3.2`, `alpine:v3.3`, `alpine:v3.4`, `alpine:v3.5`, `alpine:v3.6`, `alpine:v3.7`, `alpine:v3.8`, `alpine:v3.9`, `alpine:v3.10`, `alpine:v3.11`, `alpine:v3.12`, `alpine:v3.13`, `alpine:v3.14`, `alpine:v3.15`, `alpine:v3.16`, `alpine:v3.17`, `alpine:v3.18`, `alpine:edge`
+| `alpine:3.2`[1],`alpine:3.3`, `alpine:3.4`, `alpine:3.5`, `alpine:3.6`, `alpine:3.7`, `alpine:3.8`, `alpine:3.9`, `alpine:3.10`, `alpine:3.11`, `alpine:3.12`, `alpine:3.13`, `alpine:3.14`, `alpine:3.15`, `alpine:3.16`, `alpine:3.17`, `alpine:3.18`, `alpine:3.19` (supported in Scanner V4), `alpine:edge`
+| National Vulnerability Database (NVD)
 
 | link:https://aws.amazon.com/amazon-linux-ami[Amazon Linux]
-| `amzn:2018.03`, `amzn:2`
+| `amzn:2018.03`, `amzn:2`, `amzn:2023`[2]
+| NVD
 
-| link:https://wiki.centos.org/Manuals/ReleaseNotes[CentOS]
-| `centos:6`, `centos:7`, `centos:8`
+| CentOS
+| `centos:6`[1], `centos:7`[1], `centos:8`[1]
+| No official vulnerability feed exists.
 
 | link:https://www.debian.org/releases/[Debian]
 | `debian:10`, `debian:11`, `debian:12`, `debian:unstable`
+| NVD
+
+| link:https://www.oracle.com/linux/[Oracle Linux]
+| versions 5-9[2]
+| NVD
+
+| link:https://vmware.github.io/photon/assets/files/html/3.0/Introduction.html[Photon OS]
+| 1.0, 2.0, 3.0 [2]
+| NVD
 
 | link:https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux[{op-system-base-full}]
-| `rhel:6`, `rhel:7`, `rhel:8`, `rhel:9`
+| `rhel:6`, `rhel:7`, `rhel:8`, `rhel:9` [3]
+| Red Hat
+
+| link:https://www.suse.com/[SUSE]
+| SLES 11, 12, 15; openSUSE Leap 42.3, 15.0, 15.1; SUSE Linux [2]
+| NVD
 
 | link:http://releases.ubuntu.com/[Ubuntu]
 | `ubuntu:14.04`, `ubuntu:16.04`, `ubuntu:18.04`, `ubuntu:20.04`, `ubuntu:22.04`, `ubuntu:22.10`, `ubuntu:23.04`, `ubuntu:23.10`
+| Ubuntu CVE OVAL, NVD +
+The following vulnerability sources are not updated by the vendor:
+| `ubuntu:12.04`, `ubuntu:12.10`, `ubuntu:13.04`, `ubuntu:14.10`, `ubuntu:15.04`, `ubuntu::15.10`, `ubuntu::16.10`, `ubuntu:17.04`, `ubuntu:17.10`, `ubuntu:18.10`, `ubuntu:19.04`, `ubuntu:19.10`, `ubuntu:20.10`, `ubuntu:21.04`
 |===
+[1] Only supported in the StackRox Scanner.
+[2] Only supported in Scanner V4.
+[3] Images older than June 2020 are not be supported in Scanner V4.
```

Comment:
we used to use the `[.small]` element for these, but it's deprecated. Suggest leaving it out but doing this otherwise the way we used to:
```suggestion
|===
--
. Only supported in the StackRox Scanner.
. Only supported in Scanner V4.
. Images older than June 2020 are not be supported in Scanner V4.
--
```
I have not tested this but can in a few minutes :)